.github/workflows/trusted-release-workflow.yml

```
@@ -104,18 +105,79 @@ jobs:
+        id: checksumtxt
+        env:
+          ARTIFACTS: ${{ steps.goreleaser.outputs.artifacts }}
+        run: |
+          checksum_file=$(echo "$ARTIFACTS" | jq -r '.[] | select (.type=="Checksum") | .name')
+          echo "checksum file: ${checksum_file}"
+          echo "checksum_file=${checksum_file}" >> $GITHUB_OUTPUT
       - uses: actions/attest-build-provenance@v2
         with:
-          subject-checksums: ./dist/checksums.txt
-      # Generate attestations for checksum.txt file itself.
+          subject-checksums: ./dist/${{ steps.checksumtxt.outputs.checksum_file }}
```

actionlint (1.7.7)

input "subject-checksums" is not defined in action "actions/attest-build-provenance@v2". available inputs are "github-token", "push-to-registry", "show-summary", "subject-digest", "subject-name", "subject-path"

(action)